Boot 1089/fluxobase

moraesrebecca opened the pull request.

edimaras committed:
BOOT-926-SHARE

lucascruz77 committed:
BOOT-926-SHARE
 - Foi feito a injeçao correta do Mapstruct no teste case

lucascruz77 committed:
BOOT-926-SHARE
 - Foi feito a injeçao correta do Mapstruct no teste casa
 - Adicionado esqueleto simples para teste de Exception

lucascruz77 committed:
Merge branch 'BOOT-926-SHARE' of github.com:ebarbiericiandtier/projetct-myMusic-toTal into BOOT-926-SHARE

lucascruz77 committed:
BOOT-1089 - Fluxo base da historia 2
-> Entidade playlist (foi necessario utilizar String as ID :: tbém troquei das demais entidades
                > Para buscar via postman, devemos receber o Id como parametro, e nao consegui executar como UUID =(

-> Foi criado estrutura base da pesquisa (Controller, service, repo)
        -> PlaylistNotFoundException quando nao encontramos da playlist pelo ID recebido

moraesrebecca committed:
BOOT-1089 Adicionar musica em playlist
- Criada classe InvalidMusicException
- Alterado MusicServiceImpl, substituindo MusicNotFound por InvalidMusicException, no método findById.

moraesrebecca committed:
BOOT-1089 Adicionar música em playlist
- Alteradas rotas, passando para português, conforme documentação
- Removida rota não utilizada

moraesrebecca committed:
BOOT-1089 Adicionar música em playlist
- Alterado status code da PlaylistNotFoundException para bad request
- Removido import não utilizado

moraesrebecca committed:
BOOT-1089 Adicionar música em playlist
- Criada interface para PlaylistService
- Movida implementação de PlaylistService para impl/PlaylistServiceImpl

lucascruz77 commented on MyMusic/src/main/java/com/ciandt/summit/bootcamp2022/service/impl/MusicServiceImpl.java:
```diff
@@ -1,12 +1,15 @@
 package com.ciandt.summit.bootcamp2022.service.impl;
```
essas classes do escopo de teste devem ir pra develop mesmo?!

moraesrebecca committed:
BOOT-1089 Adicionar música em playlist
- Foram alterados os ids para String nos testes
- Foram removidos imports não usados
- Foram removidos atributos não utilizados

moraesrebecca committed:
Merge branch 'develop' of github.com:ebarbiericiandtier/projetct-myMusic-toTal into BOOT-1089/fluxobase

moraesrebecca committed:
Merge branch 'develop' into BOOT-1089/fluxobase

moraesrebecca commented:
Conflitos resolvidos

lucascruz77 committed:
BOOT-1092 - Correcao de config. no teste  unitario
